BackofficeBundle/Manager/NodeManager.php

```diff
@@ -92,6 +97,20 @@ public function createNewLanguageNode(NodeInterface $node, $language)
     }
 
     /**
+     * @param NodeInterface $node
+     *
+     * @return StatusInterface
+     */
+    protected function getInitialStatus(NodeInterface $node)
+    {
+        if ($node->getNodeType() == NodeInterface::TYPE_GENERAL) {
```

Comment:
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/718/files#diff-51685802d3e8eddde0376c8bb9d9ae88R134 on this line